src/container/cstring_view.h

```diff
@@ -19,6 +19,10 @@ namespace bpftrace {
  */
 class cstring_view : public std::string_view {
 public:
+  constexpr cstring_view() noexcept
```

Comment:
What's this needed for? It breaks `cstring_view`'s invariant of always pointing to a null-terminated string so we should try to find another way to solve the issue

Reply:
Removed - see above